src/main/java/frc/robot/constants/ShooterConstants.java

```diff
@@ -2,13 +2,53 @@
 
 import edu.wpi.first.math.controller.ArmFeedforward;
 import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
 
 public class ShooterConstants {
-    public static final double shooterP = 0.02;
-    public static final double shooterI = 0.01;
+
+    public static final int shooterTopMotorID = 13;
+    public static final int shooterBottomMotorID = 14;
+
+//    public static final double shooterP = 0.02;
+    public static final double shooterP = 0.0;
+    public static final double shooterI = 0.0;
```

Comment:
This is always good to have, although with our current time crunch and good shooter performance, lets hold off on tuning PID till after Coronado or Utah.